cogs/text_commands.py

```diff
@@ -190,118 +205,93 @@ async def cmd_command_list(self, ctx, cmd):
+        title = "Vorschlag für neuen Command Text" if command else "Vorschlag für neues Command"
+
+        embed = discord.Embed(title=title,
+                              description=f"<@!{ctx.author.id}> hat folgenden Vorschlag eingereicht.\n"
+                                          f"👍 um den Vorschlag anzunehmen\n"
+                                          f"👎 um den Vorschlag abzulehnen")
+        embed.add_field(name="\u200b", value="\u200b")
+        embed.add_field(name="Command", value=f'{cmd}', inline=False)
+        embed.add_field(name="Text", value=f'{text}', inline=False)
+        if help_message:
+            embed.add_field(name="Hilfetext", value=f'{help_message}', inline=False)
+        if category:
+            embed.add_field(name="Kategorie", value=f'{category}', inline=False)
 
         message = await mod_channel.send(embed=embed)
         await message.add_reaction("👍")
+        await message.add_reaction("👎")
         await utils.send_dm(ctx.author,
-                            "Dein Motivationstext wurde den Mods zur Genehmigung vorgelegt. Sobald er angenommen wurde, erhältst du eine Benachrichtigung.")
+                            "Dein Vorschlag wurde den Mods zur Genehmigung vorgelegt. "
+                            "Sobald darüber entschieden wurde, erhältst du eine Benachrichtigung.")
+
+    def get_or_init_command(self, cmd):
+        if command := self.text_commands.get(cmd):
+            return command
 
-    async def motivation_approved(self, message):
+        self.text_commands[cmd] = {"data": []}
+        return self.text_commands.get(cmd)
+
+    async def handle_command_reaction(self, message, approved=True):
         embed = message.embeds[0]
-        text = embed.fields[0].value
-        description = embed.description
-        ctx = await self.bot.get_context(message)
-        member_id = description[3:21]
-        guild = message.guild
-        member = await guild.fetch_member(member_id)
-
-        await utils.send_dm(member,
-                            f"Herzlichen Glückwunsch, dein Vorschlag für einen neuen Motivationstext wurde angenommen.\n\n{text}")
-        await self.cmd_add_text_command(ctx, "!motivation", text)
+        fields = {field.name: field.value for field in embed.fields}
+        cmd = fields.get("Command")
+        text = fields.get("Text")
+        help_message = fields.get("Hilfetext")
+        category = fields.get("Kategorie")
+        member = await message.guild.fetch_member(embed.description[3:21])
+
+        if approved:
+            await self.add_command(cmd, text, help_message=help_message, category=category)
+            await utils.send_dm(member,
+                                f"Herzlichen Glückwunsch, dein Vorschlag für {cmd} wurde angenommen:\n{text}")
+        else:
+            await utils.send_dm(member,
+                                f"Es tut mir leid, dein Vorschlag für {cmd} wurde leider abgelehnt:\n{text}")
```

Comment:
"Leid" braucht einen Großbuchstaben.
Abgesehen davon: Tut es Boty Leid?
*Es doppelt sich irgendwie mit dem "leider" in zweiten Halbsatz.*

Ablehnung evtl. zum Schluss positiv ergänzen, weil es sonst ein bisschen erklärungslos ist? Oder ist der Gedanke dann, dass man sich direkt als Mod beim User meldet?

Reply:
```suggestion
                                f"Dein Vorschlag für {cmd} wurde leider abgelehnt:\n{text}")
```
Wie meinst du das mit dem "zum Schluss positiv ergänzen"?

Reply:
> 
> 
> "Leid" braucht einen Großbuchstaben.
> Abgesehen davon: Tut es Boty Leid?
> _Es doppelt sich irgendwie mit dem "leider" in zweiten Halbsatz._
> 
> Ablehnung evtl. zum Schluss positiv ergänzen, weil es sonst ein bisschen erklärungslos ist? Oder ist der Gedanke dann, dass man sich direkt als Mod beim User meldet?

Wie meinst du das mit dem "zum Schluss positiv ergänzen"?

Reply:
Ich dachte an etwas, dass ein bisschen Kontext gibt? 
Bzw. dass vermeidet, dass der User o. die Userin aus Enttäuschung keine Vorschläge mehr macht.
Ich hatte vorhin keine gute Formulierung aber wie wäre es damit?

"Vielen Dank, dass du dir Gedanken darüber machst, wie man Boty mit neuen Textkommandos noch nützlicher für alle machen kann. \n
Es können nicht alle Einreichungen bestätigt werden: bspw. kann es sein, dass gerade an einer ähnlichen Idee gearbeitet wird, nur mit einer anderen Umsetzung. \n
Schreib gerne eine Direktnachricht wenn du möchtest, dass sich jemand vom Mod-Team mit dir in Verbindung setzt."
_Alternative zum letzten Satz:_ "Jemand vom Mod-Team wird sich in Verbindung mit dir setzen."